addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiControllerThingHandler.java

```diff
@@ -230,12 +235,28 @@ private void cachePut(Map<String, UniFiClient> cache, UniFiClient client) {
         }
 
         // mgb: short circuit
-        if (client == null || BooleanUtils.isNotTrue(client.isWireless()) || !belongsToSite(client, site)) {
```

Comment:
Removal of the `isWireless` check seem harmless. Just wondering why it was here in the first place. Maybe it has something to do with general support for wired devices or a left over of refactoring. @mgbowman should be able to give exclusion about this.

Reply:
LGTM as I believe it was left over from the refactoring for "future support" of wired clients.